ansible-tests/validations/network_environment.yaml

```diff
@@ -0,0 +1,14 @@
+---
+- hosts: undercloud
+  become: false
+  vars:
+    metadata:
```

Comment:
Please add the validation into a group like I did in 613bdb1fff84209212b6a2fbad989bc7783f1f05.

That's how we group validations in the mistral-based workflow.